packages/client/src/utils/getTestClient.ts

```diff
@@ -26,11 +26,9 @@ const readFile = promisify(fs.readFile)
  * Returns an in-memory client for testing
  */
 export async function getTestClient(schemaDir?: string, printWarnings?: boolean): Promise<any> {
-  if (!schemaDir) {
-    const callsite = parse(new Error('').stack!)
-    schemaDir = path.dirname(callsite[1].file!)
-  }
-  const schemaPath = await getRelativeSchemaPath(schemaDir)
+  const callSite = path.dirname(require.main?.filename ?? '')
+  const absSchemaDir = path.resolve(callSite, schemaDir ?? '')
+  const schemaPath = await getRelativeSchemaPath(absSchemaDir)
```

Comment:
Curious to know why the change was needed though it looks good

Reply:
I removed the stack trace as `require.main` should be faster. But the real change is about being able to get an in-memory client and specifying a relative path (see tests) or an absolute path. Either absolute or relative works thanks to `path.resolve`.